apps/src/sites/studio/pages/devise/registrations/_finish_sign_up.js

```diff
@@ -236,28 +237,37 @@ $(document).ready(() => {
 
   function renderSchoolInfo() {
     if (schoolInfoMountPoint) {
-      ReactDOM.render(
-        <div style={{padding: 10}}>
-          <SchoolInfoInputs
-            schoolType={schoolData.schoolType}
-            country={schoolData.country}
-            ncesSchoolId={schoolData.ncesSchoolId}
-            schoolName={schoolData.schoolName}
-            schoolCity={schoolData.schoolCity}
-            schoolState={schoolData.schoolState}
-            schoolZip={schoolData.schoolZip}
-            schoolLocation={schoolData.schoolLocation}
-            useLocationSearch={schoolData.useLocationSearch}
-            onCountryChange={onCountryChange}
-            onSchoolTypeChange={onSchoolTypeChange}
-            onSchoolChange={onSchoolChange}
-            onSchoolNotFoundChange={onSchoolNotFoundChange}
-            showRequiredIndicator={false}
-            styles={{width: 580}}
-          />
-        </div>,
-        schoolInfoMountPoint
-      );
+      if (experiments.isEnabled(experiments.SCHOOL_ASSOCIATION_V2)) {
```

Comment:
just a question, not a blocker -- how does the A/B test framework work? Will this need to be updated or does it set this experiment flag?

Reply:
That's on my sprint to investigate next! Not sure yet how it will work with Statsig, but this flag will at least allow me to test, and allow us to more easily bug bash.